### ISSUE TITLE
Update Pattern template in Figma (to have variables version) - Analytical Dashboard

### ISSUE THREAD

**Issue by yunjungyeh** [jpmorganchase/salt-ds#4267]
Update Pattern template in Figma (to have variables version) - Analytical Dashboard

**Comment by bhoppers2008** [comment]
No further updates to be made to retired libraries.

@yunjungyeh please review if this is still required for Salt DS Figma.

**Comment by bhoppers2008** [comment]
No further updates to be made to retired libraries.

@yunjungyeh please review if this is still required for Salt DS Figma.

**Comment by yunjungyeh** [comment]
Yes as we are still linking the Pattern template file from Salt DS library

**Comment by bhoppers2008** [comment]
Has a dependency on #4831. On hold until that ticket is resolved.

**Comment by bhoppers2008** [comment]
Closing issue. Further discussions need to be had about how dashboards will be supported in Salt/ DE.
